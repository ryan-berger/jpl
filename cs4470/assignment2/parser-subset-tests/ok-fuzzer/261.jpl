print ""
read image "Q>" to J
assert H9w_(), "p"
write image .923 to "Gm-Nu"
