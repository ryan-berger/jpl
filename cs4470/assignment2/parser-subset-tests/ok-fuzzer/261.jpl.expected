(PrintCmd "")
(ReadImageCmd "Q>" (VarArgument J))
(StmtCmd (AssertStmt (CallExpr H9w_) "p"))
(WriteImageCmd (FloatExpr 0) "Gm-Nu")
